File: library/src/main/java/com/getstream/sdk/chat/rest/response/QueryUserListResponse.java
package com.getstream.sdk.chat.rest.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import io.getstream.chat.android.client.models.User;

public class QueryUserListResponse {
    @SerializedName("users")
    @Expose
    private List<User> users;

    public List<User> getUsers() {
        return users;
    }
}

File: app/src/main/java/io/getstream/chat/android/core/poc/library/rest/QueryUserListResponse.kt
package io.getstream.chat.android.core.poc.library.rest

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import io.getstream.chat.android.core.poc.library.User


class QueryUserListResponse {

    @SerializedName("users")
    @Expose
    val users = listOf<User>()
}
